### ISSUE TITLE
[Suggestion] return module KIllAura+

### ISSUE THREAD

**Issue by abcdxyz6969** [MeteorClientPlus/MeteorPlus#12]
### Description

Add Module Killaura Plus Again PLSSS

### Before submitting a suggestion

- [X] This feature doesn't already exist.

- [X] This wasn't already suggested.

**Comment by Nekiplay** [comment]
Meteor+ prioritize PvE addon, it will be difficult for me to make a good killaura alone as I am not very good at algrithms

**Comment by LeonGinger** [comment]
yep, I like to use killaura for killing monsters normally, and killaura plus for killing monsters and creatures. This effectively avoids accidentally hurting objects I don't want to kill. I hope the killaura plus function back.
